workspaces/scanner/package.json

```diff
@@ -49,12 +49,12 @@
   "homepage": "https://github.com/NodeSecure/tree/master/workspaces/scanner#readme",
   "dependencies": {
     "@nodesecure/authors": "^1.0.2",
+    "@nodesecure/conformance": "^1.0.0",
```

Comment:
We did the same for tarball and tree-walker but I wonder if this should be omited ? afaik we shouldn't put ws dependencies in package.json but i may be wrong
https://docs.npmjs.com/cli/v8/using-npm/workspaces#using-workspaces

Examples in [Openally](https://github.com/OpenAlly/npm-packages/blob/main/src/ephemeral-map/src/index.ts#L13) or [Sigyn](https://github.com/MyUnisoft/sigyn/blob/main/src/agent/src/alert.ts#L2)

Reply:
As discussed in the example we bundle using tsup